agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -1223,13 +1223,72 @@ public void validate() {
     }
   }
 
+  public enum SpanFilterType {
+    REGEX
+  }
+
+  public static class SpanFilter {
+    public SpanFilterType type;
+    public String value;
+  }
+
+  public static class SpanAggregationConfig {
+    public int percentile = 95;
```

Comment:
Why this magic number?

Reply:
need to have some default percentile, 95th percentile is just an arbitrary default